pkg/controller/cluster/status.go

```diff
@@ -162,6 +162,41 @@ func (c *Controller) updateCertificatesWithRetry(ctx context.Context, tenant *mi
 	return t, nil
 }
 
+func (c *Controller) updateCustomCertificatesStatus(ctx context.Context, tenant *miniov2.Tenant, customCertificates *miniov2.CustomCertificates) (*miniov2.Tenant, error) {
+	return c.updateCustomCertificatesWithRetry(ctx, tenant, customCertificates, true)
```

Comment:
Q: As the certs status is just a report, do we need retries? Or leave it to the `syncHandler` to sync the certs status eventually? I believe there is a periodic sync enabled on the tenant objects.

Reply:
Removed retry. Synch handler deals with this, as you mention.